### ISSUE TITLE
[BUG] KeyError in Plex monitor

### ISSUE THREAD

**Issue by nauip** [iamkroot/trakt-scrobbler#155]
I run Scrobbler, an error pops up about Index.

A clear and concise description of what the bug/error is: Plex Scrobbler fails to scrobble.
Nothing had been updated from 9/7 until today when the issue mysteriously fixed itself, but I'm sending on what info I have in case it helps catch something in the future. I realize that now that the issue fixed itself this isn't highly likely.

**Desktop (please complete the following information):**
 - OS and Version: Win10 21h1
 - Python Version: 3.7.2
 - Player and Version: Plex, whatever is current

**To Reproduce**
Attempt to scrobble?

1. Open Plex 
2. Play till anything
3. Close player/pause it - This is optional
4. See error in log file - I dunno, there are several and I have to guess.
My guess is pasted below.

**Log file**

<details><summary>Click to see log contents</summary> 
<p>

```<!--DO NOT REMOVE THE THREE BACKTICKS. Replace only the next line completely-->
2021-09-22 23:20:24,989 - ERROR - plex - __init__ - Unhandled exception
Traceback (most recent call last):
  File "c:\users\nauip\.local\pipx\venvs\trakt-scrobbler\lib\site-packages\trakt_scrobbler\__init__.py", line 42, in run_with_except_hook
    run_original(*args2, **kwargs2)
  File "c:\users\nauip\.local\pipx\venvs\trakt-scrobbler\lib\site-packages\trakt_scrobbler\player_monitors\monitor.py", line 355, in run
    self.update_status()
  File "c:\users\nauip\.local\pipx\venvs\trakt-scrobbler\lib\site-packages\trakt_scrobbler\player_monitors\plex.py", line 105, in update_status
    self.status["media_info"] = self.get_media_info(status_data)
  File "c:\users\nauip\.local\pipx\venvs\trakt-scrobbler\lib\site-packages\trakt_scrobbler\player_monitors\plex.py", line 119, in get_media_info
    media_info = self._get_media_info(status_data, show_data)
  File "c:\users\nauip\.local\pipx\venvs\trakt-scrobbler\lib\site-packages\trakt_scrobbler\player_monitors\plex.py", line 136, in _get_media_info
    "episode": status_data["index"],
KeyError: 'index'
```
</p>
</details>

**Comment by iamkroot** [comment]
This seems very much a plex thing. Would be helpful to get the version number.

**Comment by iamkroot** [comment]
@nauip This fix should be included in the latest version (v1.4.0).

**Comment by iamkroot** [comment]
Closing due to inactivity.
